Neos.Flow/Classes/Http/HttpRequestHandlerInterface.php

```diff
@@ -26,15 +27,23 @@ interface HttpRequestHandlerInterface extends RequestHandlerInterface
      * Returns the currently processed HTTP request
      *
      * @return ServerRequestInterface
-     * @api
+     * @deprecated
```

Comment:
Please lets always add since when something was deprecated and why/replacement
```suggestion
     * @deprecated since 6.0 use getComponentContext()->getHttpRequest() instead
```